plugins/avro-format/core/src/main/java/org/pentaho/di/trans/steps/avro/input/AvroNestedReader.java

```diff
@@ -47,7 +47,6 @@
 import org.pentaho.di.core.variables.VariableSpace;
 import org.pentaho.di.core.vfs.KettleVFS;
 import org.pentaho.di.i18n.BaseMessages;
-import org.pentaho.di.trans.steps.avro.AvroToPdiConverter;
```

Comment:
non-blocker, just wondering why this import is removed, was it never used?

Reply:
The import is duplicated, same import is available on line no 26